js/contentAssessor.js

```diff
@@ -45,7 +50,64 @@ var ContentAssessor = function( i18n, options ) {
 	];
 };
 
-module.exports = ContentAssessor;
+require( "util" ).inherits( ContentAssessor, Assessor );
+
+ContentAssessor.prototype.calculateNegativePoints = function() {
+	var results = this.getValidResults();
+
+	var negativePoints = map( results, function( result ) {
+		var weight;
+		var rating = scoreToRating( result.getScore() );
+
+		// Convert the ratings to negative 'points'.
+		switch ( rating ) {
```

Comment:
These weight are the same for every rating. Why not declaring an array like: `{ "bad" : 1, "ok" : 0.5, "good" : 0 }` for determine the weight.